server/src/main/java/org/elasticsearch/transport/SniffConnectionStrategy.java

```diff
@@ -64,6 +64,13 @@ public class SniffConnectionStrategy extends RemoteConnectionStrategy {
         }, new StrategyValidator<>(ns, key, ConnectionStrategy.SNIFF), Setting.Property.Dynamic, Setting.Property.NodeScope)
     );
 
+    public static final Setting.AffixSetting<String> REMOTE_CLUSTER_AUTHORIZATION = Setting.affixKeySetting(
+        "cluster.remote.",
+        "authorization",
```

Comment:
Authorization was selected for a number of reasons. Options I am aware of for the value are API Key, cluster enrollment token to wrap metadata and API Key, or future proofing the name in case we want to support alternative credentials in future.Another part of future proofing is if we move Transport protocol to run over HTTP, then this matches the name of the HTTP header.

Reply:
It's a minor thing..but if we plan to mirror the HTTP header, are we also going to require the "ApiKey" prefix ?

Reply:
The feedback I got was to leave out the scheme. If we want to introduce a scheme later, I think it would be a separate setting.